packages/output/src/browser/output-widget.tsx

```diff
@@ -175,13 +173,20 @@ export class OutputWidget extends ReactWidget {
     protected getVisibleChannels(): OutputChannel[] {
         return this.outputChannelManager.getChannels().filter(channel => channel.isVisible);
     }
+
+    protected get selectedChannel(): OutputChannel | undefined {
+        return this.selectedChannelValue;
+    }
+
+    protected set selectedChannel(channel: OutputChannel | undefined) {
+        this.selectedChannelValue = channel;
+        OutputWidget.selectedChannelEmitter.fire(undefined);
+    }
 }
 
 export namespace OutputWidget {
     export namespace IDs {
-        export const CLEAR_BUTTON = 'outputClear';
```

Comment:
I don't have a problem with removing these constants to clean up, but should we add a note to the `changelog` for a breaking change? Anyone referring to these constants will be broken after the change.

Reply:
I've added an entry to the changelog.